MusicTaste/MusicTaste/AppDelegate.m

```diff
@@ -20,6 +21,12 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     
     //Parse
     
+    if ([SpotifyAPIManager shared] != nil) {
+        UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
```

Comment:
Checks if manager is running if is it will instantiate a LoginViewController when segued into it